tests/main/interfaces-network-status/task.yaml

```diff
@@ -0,0 +1,59 @@
+summary: Ensure that the network-status interface works.
+
+details: |
+    The network-status interface allows a snap to monitor networking.
+
+    A snap which defines the network-status plug must be shown in the interfaces list.
+    The plug must not be autoconnected on install and, as usual, must be able to be
```

Comment:
Why is this not auto-connected? AFAIK the network-status was a new interface designed for the sole purpose of being safe to auto-connect.

Reply:
fixed